src/app/shared/contact/contact.component.html

```diff
@@ -0,0 +1,106 @@
+<div id="contact-section">
+  <h3>Contact us</h3>
+
+  <form [formGroup]="contactForm">
+    <div class="form-group">
+      <label for="name">Full name</label>
+      <input id="name"
+             type="text"
+             class="form-control"
+             formControlName="name">
+      <span *ngIf="!contactForm.get('name').valid && contactForm.get('name').touched"
+            class="text-muted">
+        Please enter your full name!
+      </span>
+    </div>
+    <div class="form-group">
+      <label for="email">Email</label>
+      <input id="email"
+             type="text"
+             class="form-control"
+             formControlName="email">
+      <span *ngIf="!contactForm.get('email').valid && contactForm.get('email').touched"
+            class="text-muted">
+        Please enter valid email address!
+      </span>
+    </div>
+    <div class="form-group">
+      <label for="topic">Topic</label>
+      <input id="topic"
+             type="text"
+             class="form-control"
+             formControlName="topic">
+      <span *ngIf="!contactForm.get('topic').valid && contactForm.get('topic').touched"
+            class="text-muted">
+        Please enter a topic!
+      </span>
+    </div>
+    <div class="form-group">
+      <label for="message">Message</label>
```

Comment:
I am not sure, that we need to use `for` to link this control. Try to remove it. It should work without `for` keyword. Anyway, this is a tricky property and we have a risk to get a few side effects in the future